libraries/splunk_app.rb

```diff
@@ -266,6 +273,18 @@ def install_from_tar(filename, expected_version, installed_version)
         tarfile.close if tarfile
       end
 
+      def manage_lookups
+        lookups = new_resource.lookups
+        lookups.each do |file_name, url|
+          if url.nil?
```

Comment:
If we change this to `if url && !url.empty?` and flip the conditions, that means we get setting the url to false or empty string for free.